ScreenManager.lua

```diff
@@ -533,6 +528,7 @@ end
 --
 function ScreenManager.registerCallbacks( callbacks )
     local registry = {};
+    local function null() end
```

Comment:
Yes that's perfectly fine ... I'm sure using semi-colons must be horrible for any Lua coder other than me :D:D